en/docs/develop/using-the-configuration-management-rest-apis.md

```diff
@@ -20,11 +20,10 @@ met.
     Clustering](../../setup/setting-up-separate-databases-for-clustering)
     .
 
-    !!! tip
-    
-        **Note** that this REST API has been tested only with H2 and
-        MySQL5.7 databases.
+    !!! warning
     
+       This feature is enable by default for the WSO2 Identity Server 5.10.0 with the WUM update `0263`. If you do not 
```

Comment:
To use this feature, apply the **0263** WUM update for WSO2 Identity Server 5.10.0 using the WSO2 Update Manager (WUM). If you do not have this update, this feature can be enabled by running the relevant query for your database from the queries given below. 

To deploy a WUM update into production, you need to have a paid subscription. If you do not have a paid subscription, you can use this feature with the next version of WSO2 Identity Server when it is released. For more information on updating WSO2 Identity Server using WUM, see [Updating WSO2 Products](https://is.docs.wso2.com/en/5.10.0/administer/getting-wso2-updates)